frontend-react/jest.config.ts

```diff
@@ -59,6 +61,7 @@ const config: Config.InitialOptions = {
         "jest-watch-typeahead/testname",
     ],
     resetMocks: true,
+    testTimeout: IS_DEBUG ? 1000 * 60 * 5 : undefined, // Relax test timeout when debugging
```

Comment:
Might be dumb, but I'm wondering if we should just increase it to `Infinity` when we have the inspector running since it's most likely that we have breakpoints sets for debugging.  Then again, this could just cause frustration with hanging tests 😬

Reply:
Yeah I initially was going to do Infinity but figured 5 minutes was a better default and devs can tweak it locally as needed.